vortex-file/src/read/layouts/chunked.rs

```diff
@@ -77,13 +79,40 @@ impl ChunkedLayout {
         }
     }
 
+    #[allow(dead_code)]
+    fn metadata_layout(&self) -> VortexResult<Option<Box<dyn LayoutReader>>> {
+        self.has_metadata()
+            .then(|| {
+                let metadata_fb = self
+                    .flatbuffer()
+                    .children()
+                    .unwrap_or_default()
+                    .iter()
```

Comment:
flatbuffer vector derefs to slice so you can replace this with `self
                    .flatbuffer()
                    .children()
                    .ok_or_else(|| )?[0]`

Reply:
I had to use `.get(0)` but that worked.